### ISSUE TITLE
Redirect example

### ISSUE THREAD

**Issue by amagdas** [aurelia/router#82]
Hi,
It would be really useful to have in the documentation and in one of the sample apps an example of how redirects are done, it's quite hard to figure it out ATM how to do it using the latest skeleton-app.
I'm not sure how to get a reference to the app router from a different module/page/

**Comment by EisenbergEffect** [comment]
Can you provide some more concrete information for how you would like to do this? One way is to return a Redirect command from the canActivate callback of your view-model. Another way is to add a redirect property to your route config. Also, you can alway call the navigate api on a router instance. To get a Router for navigation purposes, you can have one injected into the constructor of your view-model.

**Comment by amagdas** [comment]
Thanks for the quick reply, 
Before upgrading to the latest version of the libs, I used DI to get the router but in the last code drop this was removed from the docs/sample app, this made me think that maybe this is discouraged/not a good practice in the future.
It looks like option 2 or 3 should do the trick, I want to implement a basic Login scenario with a redirect after it succeeds.
I think having some sort of documentation around the mentioned possibilities would help.

Thanks,
